public/doc/openapi-3-v1.0.0.json

```diff
@@ -1217,6 +1369,41 @@
                     }
                 }
             },
+            "LinkIn": {
+                "type": "object",
+                "description": "Input tag for creating a link",
+                "required": [
+                    "tag"
+                ],
+                "properties": {
+                    "tag": {
+                        "$ref": "#/components/schemas/Tag"
+                    }
+                }
+            },
+            "Tag": {
```

Comment:
@hsong-rh Since this refers to a Remote object should we call this RemoteObject instead of tag

Reply:
agree. In my other comments I called it `ResourceObject`. Pick one at your choice. We only need one schema. No need `LinkIn`